libsrc/common/getcpu.s

```diff
@@ -53,7 +53,7 @@ _getcpu:
         adc     #1              ; $01+$09 = $10 on 6502, $01+$09 = $0A on 2a03/2a07
         cld
         cmp     #10
```

Comment:
I am so used to seeing hexadecimal operands in ca65 source files that I was convinced that I see `$10` on that line.  I even had typed a comment (that claimed that the old code is correct) before I realized that a dollar-sign _isn't_ there.

I recommend that you change that operand to `$0A`, so that no one else will make the same mistake that I (and probably you originally) made.